core/backend/src/ViewDefinition.ts

```diff
@@ -614,9 +614,13 @@ export class TemplateViewDefinition3d extends ViewDefinition3d {
  */
 export abstract class AuxCoordSystem extends DefinitionElement {
   public static override get className(): string { return "AuxCoordSystem"; }
-  public type!: number;
+  public type?: number;
   public description?: string;
-  public constructor(props: AuxCoordSystemProps, iModel: IModelDb) { super(props, iModel); }
+  public constructor(props: AuxCoordSystemProps, iModel: IModelDb) {
+    super(props, iModel);
+    this.type = props.type;
+    this.description = props.description;
```

Comment:
For fields like these (where the type annotation already matches what the `Entity` constructor was setting), why not just use the `declare` keyword instead?

Reply:
Two things:
- Code readability. In my opinion, this is much easier to understand
- `declare` just like `!` makes code more error prone